src/components/inspector/chart-selector.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <div>
+    <label class="typo__label">Select Chart Type</label>
+    <multiselect
+      v-model="selectedOption"
+      :options="options"
```

Comment:
the order of the attributes must be, static, bindings an events

Reply:
Solved.